server/database/queries/profile.js

```diff
@@ -7,8 +7,11 @@ const clientQuery = {
     connection.query('SELECT * FROM clients ORDER BY id DESC'),
 
   // get client data
-  getClient_Data: (id) =>
-    connection.query('SELECT * FROM clients WHERE id = $1;', [id]),
+  getClient_Data: (value) =>
```

Comment:
I like this function as we said exactly.

Reply:
with the new function, I can search about the user with mobile number or Facebook or id